docs/formats/raw.md

```diff
@@ -0,0 +1,98 @@
+# Raw Format
+
+:::note
+This page describes configuration options specific to the raw data format.
+See [top-level connector documentation](/connectors/) for general information
+about configuring input and output connectors.
+:::
+
+The **raw** data format can be used to configure an input connector to pass
+raw, unparsed, data buffers from the [transport connector](/connectors/sources) to the SQL program.
+Normally the byte stream received from the transport connector is parsed using the specified data [format](/formats),
+e.g., [JSON](/formats/json) or [Avro](/formats/avro). The parser converts the raw byte stream into a stream
+of SQL table records.  In some cases it is preferable to pass the raw bytes to the SQL program unmodified and
+perform all parsing in SQL. In particular this is useful for:
+
+* **Custom data formats:** If the required data format is not supported by built-in parsers, raw data can be
+  ingested into a SQL table and processed using a [UDF](/sql/udf).
+
+* **Parallelized parsing**: Parsing large volumes of incoming data can become a performance bottleneck.
+  Offloading parsing to the SQL program enables parallel execution, leveraging the multithreaded Feldera
```

Comment:
and parallel in the near future... I would say "the parallelism in the runtime"
it is not clear now to users who does the parsing, so maybe a word about it?